main.py

```diff
@@ -2,20 +2,76 @@
 import git
 import glob
 import json
+import re
+import sys
 import os
 import shutil
 import subprocess
 
+if os.path.isdir('./cobalt/tools'):
+    sys.path.append(os.path.abspath('./cobalt/tools'))
+    import convert_dep_to_squashed
+else:
+    raise ModuleNotFoundError('third_party/depot_tools is not found')
+
+# Not perfect because it'll match a mismatched quotes
+_DEPS_FIELD_REGEX = re.compile(r'(\s*)[\'\"](src\/)?(\w+)[\'\"]\s*:')
```

Comment:
What does it match ? Would be good to have a small sample set in a test somewhere to show a couple examples.